build/buildkit/Dockerfile

```diff
@@ -7,12 +7,23 @@ RUN apk update && \
 
 FROM moby/buildkit:${BUILDKIT_TAG} AS rootless
 ARG ISTIO_GID=1337
+ARG UID=1000
 USER root
 RUN apk update && \
     apk upgrade && \
     rm -rf /var/cache/apk
+
+RUN deluser user || echo this fails for no reason && \
```

Comment:
As far as I can tell the `deluser` message is just informational and it's not failing